recipes/oommf/bld.bat

```diff
@@ -0,0 +1,5 @@
+curl -o oommf.zip  "http://math.nist.gov/oommf/dist/oommf12b0_20160930_86_x64.zip"
```

Comment:
So does that mean I can remove all the lines from the `bld.bat` file:
<pre>
curl -o oommf.zip  "http://math.nist.gov/oommf/dist/oommf12b0_20160930_86_x64.zip"

mkdir %PREFIX%\opt

7za x -o%PREFIX%\opt oommf.zip
</pre>
? I presume so and will try.